### PR TITLE
Add SciML Summer Fellowship

### PR DESCRIPTION
**Pull request by ChrisRackauckas**
@isaacsas @logankilpatrick comments?

### CODE REVIEW

**Comment by ChrisRackauckas** [comment]
Is there a way to get an email? @logankilpatrick

**Comment by isaacsas** [comment]
Looks good to me!

**Comment by logankilpatrick** [comment]
@ChrisRackauckas Do you have a google workspace account attached to Sciml.ai? We can make you a Julia one if you don't have a workspace but also NF should be able to set one up for you for free

File: dev.md [hunk 1]
```diff
@@ -8,6 +8,21 @@ Opportunities are available for everyone, all the way from students looking to
...
+of the SciML projects. Students as part of this fellowship program are expected to
+work full-time on Julia-based open source projects in the SciML organization or
+in sister organizations maintained by the SciML developers to ensure long-term
+viability for the projects. To apply, send an email to (XXXXXXXXXXXXXXX?) with a 
+CV and a two page project description, describing your prior open source contributions
+and what is planned to work on for the summer.
+
+Note that the definition of summer is left purposely vague to accomodate the differing
+schedules of university students.
+
```
Comment (logankilpatrick) [inline dev.md:25]: You might want to give people a starting place like look at the GSoC projects and use the application guidelines there (similar to what we do for JSoC)

**Comment by ChrisRackauckas** [comment]
> Do you have a google workspace account attached to Sciml.ai? We can make you a Julia one if you don't have a workspace but also NF should be able to set one up for you for free

I don't have one. How do I do this?

> Along those lines, should we have docs projects listed on the site about converting existing tutorials to use @example, and another about general improvements to the developer documentation / guides to getting started as a developer?

Yes.

**Pull request merged**